dcf/tests/init_data_for_command_test.json

```diff
@@ -0,0 +1 @@
+[{"model": "dcf.customuser", "pk": 1, "fields": {"password": "pbkdf2_sha256$30000$WAkse8FIEnVx$a/psb4ZbuuE4u7Oam9+N177gJ8QJMJuKUiAyX7j2/kY=", "last_login": "2017-05-03T03:05:34.163Z", "is_superuser": true, "username": "admin", "first_name": "", "last_name": "", "email": "admin@admin.admin", "is_staff": true, "is_active": true, "date_joined": "2017-05-03T03:05:09.422Z", "phone": null, "receive_news": true, "groups": [], "user_permissions": []}}, {"model": "dcf.customuser", "pk": 2, "fields": {"password": "pbkdf2_sha256$30000$WAkse8FIEnVx$a/psb4ZbuuE4u7Oam9+N177gJ8QJMJuKUiAyX7j2/kY=", "last_login": "2017-05-03T03:05:34.163Z", "is_superuser": true, "username": "test", "first_name": "", "last_name": "", "email": "test@admin.admin", "is_staff": true, "is_active": true, "date_joined": "2017-05-03T03:05:09.422Z", "phone": null, "receive_news": true, "groups": [], "user_permissions": []}}, {"model": "dcf.customuser", "pk": 3, "fields": {"password": "pbkdf2_sha256$30000$WAkse8FIEnVx$a/psb4ZbuuE4u7Oam9+N177gJ8QJMJuKUiAyX7j2/kY=", "last_login": "2017-05-03T03:05:34.163Z", "is_superuser": false, "username": "user", "first_name": "", "last_name": "", "email": "user@admin.admin", "is_staff": false, "is_active": true, "date_joined": "2017-05-03T03:55:09.422Z", "phone": null, "receive_news": true, "groups": [], "user_permissions": []}}, {"model": "dcf.section", "pk": 1, "fields": {"title": "test"}}, {"model": "dcf.group", "pk": 1, "fields": {"slug": "mygroup", "title": "mygroup", "section": 1}}, {"model": "dcf.item", "pk": 1, "fields": {"slug": "item_1", "user": 1, "group": 1, "title": "Item #1", "description": "Item #1", "price": "111.00", "phone": "8999999999", "is_active": true, "updated": "2017-04-28T03:06:45.462Z", "posted": "2017-05-03T03:06:45.462Z"}}, {"model": "dcf.item", "pk": 2, "fields": {"slug": "item_2", "user": 1, "group": 1, "title": "Item #2", "description": "Item #2", "price": "1212.00", "phone": "8999999999", "is_active": true, "updated": "2017-05-03T03:07:21.797Z", "posted": "2017-05-03T03:07:21.797Z"}}, {"model": "dcf.item", "pk": 3, "fields": {"slug": "item_3", "user": 1, "group": 1, "title": "Item #3", "description": "Item #3", "price": "1212.00", "phone": "8999999999", "is_active": true, "updated": "2017-04-03T03:07:21.797Z", "posted": "2017-04-03T03:07:21.797Z"}}, {"model": "dcf.item", "pk": 4, "fields": {"slug": "item_4", "user": 2, "group": 1, "title": "Item #4", "description": "Item #4", "price": "111.00", "phone": "8999999999", "is_active": true, "updated": "2017-04-28T03:06:45.462Z", "posted": "2017-05-03T03:06:45.462Z"}}, {"model": "dcf.item", "pk": 5, "fields": {"slug": "item_5", "user": 3, "group": 1, "title": "Item #5", "description": "Item #5", "price": "10.00", "phone": "822222222", "is_active": true, "updated": "2016-06-28T03:06:45.462Z", "posted": "2016-05-03T03:06:45.462Z"}}]
```

Comment:
You can do that either creating inactive items or just mark some test items as inactive after you created them.